src/ripple/app/tests/MultiSign.test.cpp

```diff
@@ -741,6 +962,19 @@ class MultiSign_test : public beast::unit_test::suite
                     "fails local checks: Invalid signature.");
         }
         {
+            // Single-sign, but invalidate the transaction.
```

Comment:
yes, that comment is wonky. This is very similar to the existing test above it, so I'm inclined to leave them together.